service/keyv2/key.go

```diff
@@ -105,10 +105,22 @@ func LoadBalancerName(domainName string, cluster v1alpha1.AWSConfig) (string, er
 	return lbName, nil
 }
 
-func MainStackName(customObject v1alpha1.AWSConfig) string {
+func MainGuestStackName(customObject v1alpha1.AWSConfig) string {
 	clusterID := ClusterID(customObject)
 
-	return fmt.Sprintf("%s-main", clusterID)
+	return fmt.Sprintf("%s-guest-main", clusterID)
+}
+
+func MainHostPreStackName(customObject v1alpha1.AWSConfig) string {
+	clusterID := ClusterID(customObject)
+
+	return fmt.Sprintf("%s-host-pre-main", clusterID)
```

Comment:
This had to be split but I'm not sure the pre and post names are intuitive. What about if we called the pre-stack `%s-host-setup` and the post stack `%s-host-main`?

Maybe in the stack descriptions we can say the stacks need to run pre and post the guest stack?

Reply:
Sure, much better, done